test/Npgsql.Tests/AutoPrepareTests.cs

```diff
@@ -316,6 +316,44 @@ public void OverloadedSql()
             }
         }
 
+#if !NETCOREAPP1_1
```

Comment:
Any specific reason not to run this test on NETCOREAPP1_1?

Reply:
Yes, NETCOREAPP1_1 doesn't have NpgsqlCommandBuilder.

Reply:
Good answer.